docs/versioned_docs/version-2.11.0/concepts/jiva.md

```diff
@@ -1,6 +1,9 @@
 ---
 id: jiva
 title: Jiva Overview
+keywords: 
+  - Jiva
```

Comment:
Pls add OpenEBS & OpenEBS comunity as keywords here.

Reply:
SEO keywords are the words and phrases that searchers enter into search engines and these help them land on the correct page. IMO, `OpenEBS community` will be the right keyword for only the community page.